src/urllib3/__init__.py

```diff
@@ -20,6 +20,19 @@
 from .util.retry import Retry
 from .util.timeout import Timeout
 
+try:
```

Comment:
```suggestion
# note to vendors and OS distributors delete this block
try: 
```

Reply:
@graingert  I don't understand your question. What does it mean `we don't want pip itself to hit this`? 
What this block of code doing here is to try importing the marker package `urllib3-secure-extra` (which indicate `secure` extra has installed), if import success, we raise the `DeprecationWarning` message.

Reply:
so urllib3 is pasted wholesale into `pip._vendor` and we don't want installing a package in someone's virtual environment to start causing pip issuing warnings

Additionally some distributors like RedHat/fedora/canonical/debian will redistribute urllib3 and they might not want it issuing warnings to their users. We certainly don't want them to attempt to distribute the marker package.

Doubtless they'll make the right choice here even if we don't add a note here

Reply:
Excellent idea, thanks @graingert!